lib/Metadata.php

```diff
@@ -44,6 +44,11 @@ class Metadata
     private $fieldMappings = [];
 
     /**
+     * @var bool
+     */
+    private $removeLive = true;
```

Comment:
Not completely sure about this now, but `removeLive` could also be interpreted as never having a live node at all.